### DISCUSSION TITLE
Usage of Checkbox Group

### DISCUSSION THREAD

**Discussion by RBilly** [adobe/react-spectrum#4563] [Q&A] [answered]
Hello !
I'm prototyping a design system and I'm playing around with react-aria components.
I love react-aria hooks but they are a bit complex to use when trying to prototype and react-aria, component, while still in alpha, are a great ! Even better than radix-ui from my perspective. 
So I wanted to say thank you to the authors and contributors, I admire your work.

Now to the subject that led me to create this discussion, I encountered an issue in Storybook while trying to display a group of checkbox. Nothing fancy, a CheckboxGroup with a Label and 3 Checkbox inside. 
I probably did something wrong because the state of all 3 checkbox seems to be linked (when I select one, the three are selected).
Does someone has any idea as to why it happens ?

Thanks for your help !

**Comment by snowystinger** [comment, marked as answer]
Hey, thanks for the compliments! It means a lot to us. I'm sorry to hear that the CheckboxGroup isn't working for you. Can you make a Codesandbox demonstrating the issue? Here is one with a CheckboxGroup to get you started https://codesandbox.io/s/naughty-mcnulty-bteeg3?file=/src/App.js I'm not seeing the issue here, so we'll just need to find out what is different between this and your implementation.

I think my first hunch would be non-unique `value` props on the Checkboxes, or missing the `value` prop, which would lead to the same issue.

**Comment by RBilly** [comment]
Oh wow I feel so dumb now...
My apologies !
I copy/pasted my checkbox exemple which didn't have any value prop without noticing that it was mandatory inside a CheckboxGroup.
I'm very sorry for taking your time for nothing. I should have paid more attention while reading the doc.

Thanks for the quick reply and keep up the good work !
